bot/cogs/dogeboard.py

```diff
@@ -45,103 +35,65 @@ def __init__(self, bot: Friendo) -> None:
-        if self._token:
-            return {"Authorization": f"Bearer {self._token}"}
-
-    @staticmethod
-    async def _handle_request(resp: ClientResponse) -> Dict[str, Any]:
-        """Handle the request errors and status codes."""
-        if resp.status != 200:
-            raise ConnectionError(f"Error: {resp.status} | Cannot connect to GraphQL Database")
-
-        data = await resp.json()
-        if errors := data.get("errors"):
-            raise QueryError([error["message"] for error in errors])
-        return data
-
-    async def _login(self) -> None:
-        """Login to the GraphQL database and store the token."""
-        query = """mutation {
-          login(data: { username: "%s", password: "%s" }) {
-            token
-          }
-        }
-        """ % (settings.FRIENDO_API_USER, settings.FRIENDO_API_PASS)
-
-        async with self.bot.session.post(self._url, json={"query": query}) as resp:
-            data = await self._handle_request(resp)
-            try:
-                self._token = data["data"]["login"]["token"]
-            except QueryError as e:
-                log.error(f"Couldn't login: {e}")
-
     async def _get_guild_data(self, guild_id: int) -> Optional[DogeBoardData]:
         """
         Attempt to get the dogeboard_data from cache using guild_id.
 
         if the data is not in the cache, it will request it from the GraphQL DB.
         if the data doesn't exist in the database it will be created and send to GraphQL DB
         """
-        if self._token is None:
-            await self._login()
-
         if dogeboard_data := self._cache.get(guild_id):
             return dogeboard_data
         elif dogeboard_data := await self._fetch_guild_data(guild_id):
             return dogeboard_data
 
     async def _fetch_guild_data(self, guild_id: int) -> Optional[DogeBoardData]:
         """Attempt to fetch guild data from the GraphQL database."""
-        query = """mutation {
-            get_guild(data: { guild_id: "%s", }) {
-                guild_id
-                dogeboard_id
-                dogeboard_emoji
-                dogeboard_reactions_required
-              }
-        }""" % guild_id
-
-        async with self.bot.session.post(self._url, json={"query": query}, headers=self.headers) as resp:
-            try:
-                data = await self._handle_request(resp)
-
-                dogeboard_data = DogeBoardData(**data["data"]["get_guild"])
-                self._cache[guild_id] = dogeboard_data
-                return dogeboard_data
-            except QueryError as e:
-                log.error(f"Query Errors: {e}")
+        query = (
+            "mutation fetch_guild($guild_id: BigInt!) {"
+            "   get_guild(data: { guild_id: $guild_id }) {"
+            "    guild_id"
+            "    dogeboard_id"
+            "    dogeboard_emoji"
+            "    dogeboard_reactions_required"
+            "  }"
+            "}"
+        )
```

Comment:
what if like, multi-line strings via double `"""` and then some `textwrap.dedent`?